aa_stripe/forms.py

```diff
@@ -5,6 +5,14 @@
 
 
 class StripeCouponForm(forms.ModelForm):
+    def clean_currency(self):
```

Comment:
Please remove this, and simply set this not blank in admin. Every coupon requires currency.

Reply:
If "percent_off" is specified, currency is not needed, but as we have "default" and there's a list of choices I can make this required in admin, this seems okay.

Reply:
Added in the CR commit